README.md

```diff
@@ -1,14 +1,120 @@
+## Running the service
+
+*From the root directory*
+
+`sbt run` - starts the service locally
+
+`sbt runAllChecks` - Will run all checks required for a successful build
+
+Default service port on local - 9840
+
+### Required dependencies
+
+There are a number of dependencies required to run the service.
+
+The easiest way to get started with these is via the service manager CLI - you can find the installation guide [here](https://docs.tax.service.gov.uk/mdtp-handbook/documentation/developer-set-up/set-up-service-manager.html)
+
+| Command                                       | Description                                                      |
+|-----------------------------------------------|------------------------------------------------------------------|
+| `sm2 --start CUSTOMS_FINANCIALS_ALL`          | Runs all dependencies                                            |
+| `sm2 -s`                                      | Shows running services                                           |
+| `sm2 --stop CUSTOMS_FINANCIALS_SESSION_CACHE` | Stop the micro service                                           |
+| `sbt run` or ` sbt "run 9840"`                | (from root dir) to compile the current service with your changes |
+
+### Runtime Dependencies
+(These are subject to change and may not include every dependency)
+
+* `AUTH`
+* `AUTH_LOGIN_STUB`
+* `AUTH_LOGIN_API`
+* `BAS_GATEWAY`
+* `CA_FRONTEND`
+* `SSO`
+* `USER_DETAILS`
+
+### Login enrolments
+
+The service can be accessed by using below enrolments and with below sample EORI numbers, via http://localhost:9949/auth-login-stub/gg-sign-in (on local) or https://<host:port>/auth-login-stub/gg-sign-in on DEV/QA/STAGING
+
+Redirect URL - `/customs/payment-records`
+
+| Enrolment Key	 | Identifier Name | Identifier Value |
+|----------------|-----------------|------------------|
+| `HMRC-CUS-ORG` | `EORINumber`    | `GB744638982000` |
+| `HMRC-CUS-ORG` | `EORINumber`    | `GB744638982001` |
+
+## Testing
 
+The minimum requirement for test coverage is 90%. Builds will fail when the project drops below this threshold.
+
+### Unit Tests
+
+| Command                                | Description                  |
+|----------------------------------------|------------------------------|
+| `sbt test`                             | Runs unit tests locally      |
+| `sbt "test/testOnly *TEST_FILE_NAME*"` | runs tests for a single file |
+
+### Coverage
+
+| Command                                  | Description                                                                                                 |
+|------------------------------------------|-------------------------------------------------------------------------------------------------------------|
+| `sbt clean coverage test coverageReport` | Generates a unit test coverage report that you can find here target/scala-3.3.5/scoverage-report/index.html |
+
+
+## Feature Switches
```

Comment:
I think better to add just 'Not applicable' under Feature Switches section (and remove other details) as its better to explicitly mention that there are no feature flags for this service